01-node-tutorial/11-fs-async.js

```diff
@@ -26,4 +26,4 @@ readFile('./content/first.txt', 'utf8', (err, result) => {
     )
   })
 })
```

Comment:
Fill like in this function you are writing in a different files instead of in the demanded file `./temporary/fileB.txt` that is why they have instructed you to use  the append flag  that allow you to be able to write to the next line